### ISSUE TITLE
Question: auto formatting & sorting imports

### ISSUE THREAD

**Issue by h4l** [jazzband/docopt-ng#39]
@NickCrews would you be OK with auto sorting/formatting imports (probably with [isort](https://pycqa.github.io/isort/))? 

When I was rebasing the error message PR just now, and a few other branches I've got, probably 3/4 of my merge conflicts were just imports. I'm thinking it would help to use isort with the single line option, so that instead of:

```python
from typing import Any, Callable, Tuple, Type, Union, cast
```

we get:
```python
from typing import Any
from typing import Callable
from typing import Tuple
from typing import Type
from typing import Union
from typing import cast
```

Which doesn't look as nice, but means import changes shouldn't cause merge conflicts.

**Comment by h4l** [comment]
This is what it would look like: https://github.com/jazzband/docopt-ng/compare/master...h4l:docopt-ng:isort

**Comment by NickCrews** [comment]
Yes I LOVE autoformatters such as black and I sort. Please implement that of you like, or I can do it too.

**Comment by h4l** [comment]
Awesome! Yep, please do.

**Comment by NickCrews** [comment]
I merged #37 first to avoid merge conflicts, which then made your branch have conflicts, so I just re-implemented it myself

**Comment by h4l** [comment]
Thanks @NickCrews, that's sensible. Good to move towards pyproject.toml too.
